VideoLocker/src/main/java/org/edx/mobile/module/analytics/ISegmentImpl.java

```diff
@@ -44,6 +50,21 @@ public void setAppNameContext() {
             this.properties.put(Keys.CONTEXT, getAppNameContext());
         }
 
+        public void setCustomDimensions() {
```

Comment:
This method should have a more generic name (`setGlobalState()`?), as it's setting the navigation mode as well as the custom dimensions.